react/public/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "PingOne for Customers Custom SignOn React Client App",
```

Comment:
Do these names need to change?

Reply:
Please see my answer above(the same cause)